src/mlpack/methods/logistic_regression/logistic_regression_main.cpp

```diff
@@ -75,7 +75,9 @@ PROGRAM_INFO("L2-regularized Logistic Regression and Prediction",
     "so long as an existing logistic regression model is given with the " +
     PRINT_PARAM_STRING("input_model") + " parameter.  The output predictions "
     "from the logistic regression model may be saved with the " +
-    PRINT_PARAM_STRING("output") + " parameter."
+    PRINT_PARAM_STRING("predictions") + " parameter." + 
+    "The" + PRINT_PARAM_STRING("output") + "parameter is depracated" + 
```

Comment:
deprecated :)

Also I would suggest putting this in its own paragraph (so a `\n\n` before the `The`), and we should point out that `output_probabilities` is also deprecated.

Reply:
should I also mention the respective replacements for the deprecated parameters in the paragraph ?